index.html

```diff
@@ -1,23 +1,491 @@
+                                <img src="/image/Negative3.png" alt="">
+                            </div>
+                            <div class="authors__mxh--twicon">
+                                <img src="/image/Negative.png" alt="">
+                            </div>
+                            <div class="authors__mxh--inicon">
+                                <img src="/image/Negative2.png" alt="">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="authors__item">
+                        <div class="authors__img">
+                            <img src="./image/pexels-vazhnik-7562313.jpg" alt="">
+                        </div>
+                        <div class="authors__name">Floyd Miles</div>
+                        <div class="authors__about">Content Writer @Company</div>
+                        <div class="authors__mxh">
+                            <div class="authors__mxh--fbicon">
+                                <img src="/image/Negative4.png" alt="">
+                            </div>
+                            <div class="authors__mxh--igicon">
+                                <img src="/image/Negative3.png" alt="">
+                            </div>
+                            <div class="authors__mxh--twicon">
+                                <img src="/image/Negative.png" alt="">
+                            </div>
+                            <div class="authors__mxh--inicon">
+                                <img src="/image/Negative2.png" alt="">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="authors__item">
+                        <div class="authors__img">
+                            <img src="./image/pexels-vazhnik-7562313.jpg" alt="">
+                        </div>
+                        <div class="authors__name">Floyd Miles</div>
+                        <div class="authors__about">Content Writer @Company</div>
+                        <div class="authors__mxh">
+                            <div class="authors__mxh--fbicon">
+                                <img src="/image/Negative4.png" alt="">
+                            </div>
+                            <div class="authors__mxh--igicon">
+                                <img src="/image/Negative3.png" alt="">
+                            </div>
+                            <div class="authors__mxh--twicon">
+                                <img src="/image/Negative.png" alt="">
+                            </div>
+                            <div class="authors__mxh--inicon">
+                                <img src="/image/Negative2.png" alt="">
+                            </div>
+                        </div>
+                    </div>
+
+                </div>
+            </div>
+
+        </div>
+
+        <!--  -->
+        <div class="component">
```

Comment:
Cái phần này có mỗi homepage có thì làm sao gọi là component được
```suggestion
        <div class="home__featured-in">
```